.github/.jira_sync_config.yaml

```diff
@@ -0,0 +1,28 @@
+settings:
+  jira_project_key: "DOCPR"
+  epic_key: "DOCPR-2376"
+  components:
+    - Starter Pack
+  labels:
+    - "bug"
+    - "documentation"
+    - "enhancement"
+    - "improvement"
+    - "question"
+  status_mapping:
+    opened: Untriaged
```

Comment:
It might be worth making this `Triaged`. As it stands we'll need to triage twice: first for a label, and second for the Jira task.

Alternatively, we could add a `triaged` label as the trigger, to separate those concerns.